README.md

```diff
@@ -69,6 +69,12 @@ system = forcefield.createSystem(topology, mols)
 See `examples/SMIRNOFF_simulation/` for a complete example of how SMIRNOFF can be used for small molecule vacuum simulations, and `examples/mixedFF_structure` for how to set up a system which uses an AMBER forcefield (in this case, AMBER99SB-ILDN) for a protein in combination with SMIRNOFF for a small molecules. Via ParmEd, this can be translated into GROMACS, AMBER, or CHARMM formats for use elsewhere (and additional formats via InterMol).
 **For an especially complete worked example, see complete setup of a host-guest simulation in water with SMIRNOFF**, including docking, 2D and 3D visualization, etc., in `examples/host_guest_simulation/smirnoff_host_guest.ipynb` (Jupyter notebook).
 
+### FAQs:
+See `Chemical-starting-points.md` for answers to a variety of common problems, such as:
```

Comment:
Can you add a link to the file so people can click on it? Or do we have to add that in a separate PR when that file is in the repo?

Reply:
Can add, yes.